packages/fast-tooling-react/src/css-editor/border/border.props.ts

```diff
@@ -2,6 +2,7 @@ import { ManagedClasses } from "@microsoft/fast-jss-manager-react";
 import { CSSBorderClassNameContract } from "./border.style";
 
 export enum BorderStyleValue {
+    "" = "",
```

Comment:
This seems kinda strange, why are you having to do this?

Reply:
This is for the default / nothing selected state